File: backend/src/main/java/com/ll/TeamProject/domain/calendar/entity/Message.java
package com.ll.TeamProject.domain.calendar.entity;

import com.ll.TeamProject.domain.user.entity.SiteUser;
import com.ll.TeamProject.global.jpa.entity.BaseTime;
import jakarta.persistence.Entity;
import jakarta.persistence.FetchType;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import lombok.Getter;

@Entity
@Getter
public class Message extends BaseTime {
    // BaseTime : id (BaseEntity, no setter), 생성/수정일

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "user_id") // 작성자
    private SiteUser user;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "calendar_id")
    private Calendar calendar;

    private String content; // 메시지 내용

    private boolean isRead; // 읽음 여부
}

File: backend/src/main/kotlin/com/ll/TeamProject/domain/calendar/entity/Message.kt
package com.ll.TeamProject.domain.calendar.entity

import com.ll.TeamProject.domain.user.entity.SiteUser
import com.ll.TeamProject.global.jpa.entity.BaseTime
import jakarta.persistence.*
import lombok.Getter

@Entity
@Getter
class Message(
    @field:ManyToOne(fetch = FetchType.LAZY)
    @field:JoinColumn(name = "user_id") // 작성자
    var user: SiteUser,

    @field:ManyToOne(fetch = FetchType.LAZY)
    @field:JoinColumn(name = "calendar_id")
    var calendar: Calendar,

    var content: String, // 메시지 내용

    var isRead: Boolean // 읽음 여부
) : BaseTime() {
    // BaseTime은 id (BaseEntity, no setter), 생성/수정일 포함
}